lightning/src/util/events.rs

```diff
@@ -602,6 +620,41 @@ pub enum Event {
 		/// Destination of the HTLC that failed to be processed.
 		failed_next_destination: HTLCDestination,
 	},
+	/// Indicates that a channel featuring anchors outputs is to be closed by broadcasting the local
+	/// commitment transaction. Since commitment transactions have a static feerate pre-agreed upon,
+	/// they may need additional fees to be attached through a child transaction using the popular
+	/// Child-Pays-For-Parent (CPFP) fee bumping technique. This child transaction must include the
+	/// anchor input found within `anchor_descriptor` along with additional inputs to meet the
+	/// target feerate. Once the transaction is constructed, it must be fully signed for and
```

Comment:
"Failure to meet the target feerate decreases the confirmation odds of this transaction detected as time-sensitive, this could lead to loss of funds" Like to make clearer meeting the target is a MUST?